src/internal/pjs/server.go

```diff
@@ -1,14 +1,25 @@
 package pjs
 
 import (
+	"context"
+
+	"github.com/pachyderm/pachyderm/v2/src/auth"
 	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
+
 	pjsserver "github.com/pachyderm/pachyderm/v2/src/pjs"
 )
 
 type Env struct {
-	DB *pachsql.DB
+	DB        *pachsql.DB
+	Permitter Permitter
 }
 
 func NewAPIServer(env Env) pjsserver.APIServer {
 	return newAPIServer(env)
 }
+
+// Permitter is an interface that currently exposes the Pachyderm Auth's server's GetPermissions RPC.
```

Comment:
“Pachyderm auth server’s”

Arguably this would be more idiomatically named `PermissionGetter`, or even `GetPermissioner`, ugly as it is.